viewer/views.py

```diff
@@ -364,6 +381,42 @@ def get_serializer_class(self):
             return SnapshotReadSerializer
         return SnapshotWriteSerializer
     filter_class = filters.SnapshotFilter
-    # filter_permissions = "target_id__project_id"
-    # filter_fields = '__all__'
 ### End of Session Project
+
+
+class TargetCompoundSetsView(views.APIView):
+    def get(self, request, targetID):
+        target = Target.objects.get(id=targetID)
+        compound_sets = CompoundSet.objects.filter(target=target)
+        a_compound_set = compound_sets[0]
```

Comment:
This will just return the first compound set from the query. We need to return all of them, and then add each one as a tab on the front-end

Reply:
I dont use this API method.....thanks for review I'm going to remove given method